web/modules/custom/dpl_example_content/content/node/dac275e4-9b8c-4959-a13a-6b9fdbc1f6b0.yml

```diff
@@ -0,0 +1,84 @@
+_meta:
+  version: '1.0'
+  entity_type: node
+  uuid: dac275e4-9b8c-4959-a13a-6b9fdbc1f6b0
+  bundle: branch
+  default_langcode: da
+  depends:
+    e00e1460-169e-468d-9835-d664f06d0969: file
+    7b9d1894-7f9a-4f96-a7dc-efa3af5fc902: media
+default:
+  revision_uid:
+    -
+      target_id: 1
+  status:
+    -
+      value: true
+  uid:
+    -
+      target_id: 1
+  title:
+    -
+      value: Hovedbiblioteket
+  created:
+    -
+      value: 1708940972
+  promote:
+    -
+      value: false
+  sticky:
+    -
+      value: false
+  revision_translation_affected:
+    -
+      value: true
+  entity_clone_template_image:
+    -
+      entity: e00e1460-169e-468d-9835-d664f06d0969
+      alt: ''
+      title: ''
+      width: 0
+      height: 0
+  metatag:
+    -
+      tag: meta
+      attributes:
+        name: title
+        content: 'Hovedbiblioteket | DPL CMS'
+  field_address:
+    -
+      langcode: da
+      country_code: DK
+      locality: København
+      postal_code: '1172'
+      address_line1: 'Krystalgade 15'
+      address_line2: ''
```

Comment:
Consider for at least one of the examples to use the [data from Figma](https://www.figma.com/file/Zx9GrkFA3l4ISvyZD2q0Qi/Designsystem?type=design&node-id=7486-64797&mode=design&t=HChMdePrpYExycVR-4) where possible.

This makes it easy for us to see how what we have implemented corresponds to what was originally proposed. Note that images are a problem here due to copyright.

Reply:
You mean specifically `address_line2`, or?

Reply:
No - the address in general.

Reply:
I've updated it, so it also includes the brønshøj address